nnet_utils/nnet_activation.h

```diff
@@ -137,9 +137,15 @@ template<class data_T, class res_T, typename CONFIG_T>
 void  sigmoid(data_T data[CONFIG_T::n_in], res_T res[CONFIG_T::n_in])
 {
     // Initialize the lookup table
-    static typename CONFIG_T::table_t sigmoid_table[CONFIG_T::table_size] ={-999.};
-    if(sigmoid_table[0] == -999.)
+    static bool initialized = false;
+    static typename CONFIG_T::table_t sigmoid_table[CONFIG_T::table_size];
+    if (!initialized)
+    {
       init_sigmoid_table<CONFIG_T, CONFIG_T::table_size>(sigmoid_table);
+      initialized = true;
+    }
+
+    if(sigmoid_table[0] == CONFIG_T::table_dummy)
```

Comment:
What is the purpose of this line?

Reply:
Ops, crap, this line should be removed. Let me fix it! Thanks